.github/workflows/claude-nl-suite.yml

```
@@ -218,80 +343,283 @@ jobs:
+              Stop condition:
+              - After T-J_results.xml is written, stop.
+            timeout_minutes: "30"
+            anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+
+        # (moved) Assert T coverage after staged fragments are promoted
+
+        - name: Retry T pass (Sonnet) if incomplete
+          if: failure()
+          uses: anthropics/claude-code-base-action@beta
+          with:
+            use_node_cache: false
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)